### PR TITLE
Remove @Autowired (#3926)

### PR DESCRIPTION
**Pull request by freddidierRTE**
Nothing in release note

### CODE REVIEW

File: services/cards-consultation/src/main/java/org/opfab/cards/consultation/services/RabbitMQEntryPoint.java [hunk 1]
```diff
@@ -52,7 +51,7 @@ public class RabbitMQEntryPoint {
     private SimpleMessageListenerContainer ackListener;
 
 
-    @Autowired
+
     public RabbitMQEntryPoint(AmqpAdmin amqpAdmin,
             FanoutExchange cardExchange,
             FanoutExchange processExchange,
```
Comment (quinarygio) [inline services/cards-consultation/src/main/java/org/opfab/cards/consultation/services/RabbitMQEntryPoint.java:57]: Fix Copyright

File: services/cards-publication/src/test/java/org/opfab/cards/publication/configuration/TestCardReceiver.java [hunk 2]
```diff
@@ -7,37 +7,27 @@
  * This file is part of the OperatorFabric project.
  */
```
Comment (quinarygio) [inline services/cards-publication/src/test/java/org/opfab/cards/publication/configuration/TestCardReceiver.java:9]: Fix Copyright

File: services/external-devices/src/test/java/org/opfab/externaldevices/drivers/ModbusDriverFactoryShould.java [hunk 3]
```diff
@@ -46,7 +46,7 @@ void modbusDriverFactoryShouldCreateDriverForResolvableHost() throws ExternalDev
 
     @Test
     void modbusDriverFactoryShouldThrowErrorIfAttemptingToCreateDriverWithUnresolvableHost() {
-        assertThrows(ExternalDeviceDriverException.class, () -> {modbusDriverFactory.create("unresolvableHost",123);});
+        assertThrows(ExternalDeviceDriverException.class, () -> {modbusDriverFactory.create("unresolvable Host",123);});
```
Comment (quinarygio) [inline services/external-devices/src/test/java/org/opfab/externaldevices/drivers/ModbusDriverFactoryShould.java:49]: Fix Copyright

File: services/users/src/main/java/org/opfab/users/configuration/oauth2/WebSecurityChecks.java [hunk 4]
```diff
@@ -36,9 +36,12 @@
 @Slf4j
```
Comment (quinarygio) [inline services/users/src/main/java/org/opfab/users/configuration/oauth2/WebSecurityChecks.java:36]: Fix Copyright

File: tools/spring/spring-mongo-utilities/src/main/java/org/opfab/springtools/configuration/mongo/MongoConfiguration.java [hunk 5]
```diff
@@ -46,7 +46,6 @@ public class MongoConfiguration extends AbstractReactiveMongoConfiguration {
```
Comment (quinarygio) [inline tools/spring/spring-mongo-utilities/src/main/java/org/opfab/springtools/configuration/mongo/MongoConfiguration.java:46]: Fix Copyright

File: tools/spring/spring-oauth2-utilities/src/main/java/org/opfab/springtools/configuration/oauth/I18nProcessesCache.java [hunk 6]
```diff
@@ -11,7 +11,6 @@
```
Comment (quinarygio) [inline tools/spring/spring-oauth2-utilities/src/main/java/org/opfab/springtools/configuration/oauth/I18nProcessesCache.java:11]: Fix Copyright

File: tools/spring/spring-oauth2-utilities/src/main/java/org/opfab/springtools/configuration/oauth/OAuth2GenericConfiguration.java [hunk 7]
```diff
@@ -19,7 +19,6 @@
 import org.opfab.springtools.configuration.oauth.jwt.groups.GroupsUtils;
 import org.opfab.users.model.CurrentUserWithPerimeters;
 import org.opfab.users.model.User;
-import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.cache.annotation.EnableCaching;
 import org.springframework.cloud.openfeign.EnableFeignClients;
 import org.springframework.context.annotation.Bean;
```
Comment (quinarygio) [inline tools/spring/spring-oauth2-utilities/src/main/java/org/opfab/springtools/configuration/oauth/OAuth2GenericConfiguration.java:24]: Fix Copyright

File: tools/spring/spring-oauth2-utilities/src/main/java/org/opfab/springtools/configuration/oauth/OAuth2ReactiveConfiguration.java [hunk 8]
```diff
@@ -11,6 +11,9 @@
```
Comment (quinarygio) [inline tools/spring/spring-oauth2-utilities/src/main/java/org/opfab/springtools/configuration/oauth/OAuth2ReactiveConfiguration.java:11]: Fix Copyright

File: tools/spring/spring-oauth2-utilities/src/main/java/org/opfab/springtools/configuration/oauth/jwt/groups/GroupsUtils.java [hunk 9]
```diff
@@ -14,7 +14,6 @@
 import java.util.stream.Collectors;
```
Comment (quinarygio) [inline tools/spring/spring-oauth2-utilities/src/main/java/org/opfab/springtools/configuration/oauth/jwt/groups/GroupsUtils.java:14]: Fix Copyright

**Review by quinarygio — CHANGES_REQUESTED** [review]

File: services/cards-consultation/src/main/java/org/opfab/cards/consultation/services/RabbitMQEntryPoint.java (hunk 1 above)
Comment (freddidierRTE) [reply services/cards-consultation/src/main/java/org/opfab/cards/consultation/services/RabbitMQEntryPoint.java:57]: Fixed

File: services/cards-publication/src/test/java/org/opfab/cards/publication/configuration/TestCardReceiver.java (hunk 2 above)
Comment (freddidierRTE) [reply services/cards-publication/src/test/java/org/opfab/cards/publication/configuration/TestCardReceiver.java:9]: Fixed

File: services/external-devices/src/test/java/org/opfab/externaldevices/drivers/ModbusDriverFactoryShould.java (hunk 3 above)
Comment (freddidierRTE) [reply services/external-devices/src/test/java/org/opfab/externaldevices/drivers/ModbusDriverFactoryShould.java:47]: Fixed

File: services/users/src/main/java/org/opfab/users/configuration/oauth2/WebSecurityChecks.java (hunk 4 above)
Comment (freddidierRTE) [reply services/users/src/main/java/org/opfab/users/configuration/oauth2/WebSecurityChecks.java:35]: Fixed

File: tools/spring/spring-mongo-utilities/src/main/java/org/opfab/springtools/configuration/mongo/MongoConfiguration.java (hunk 5 above)
Comment (freddidierRTE) [reply tools/spring/spring-mongo-utilities/src/main/java/org/opfab/springtools/configuration/mongo/MongoConfiguration.java:46]: Fixed

File: tools/spring/spring-oauth2-utilities/src/main/java/org/opfab/springtools/configuration/oauth/I18nProcessesCache.java (hunk 6 above)
Comment (freddidierRTE) [reply tools/spring/spring-oauth2-utilities/src/main/java/org/opfab/springtools/configuration/oauth/I18nProcessesCache.java:11]: Fixed

File: tools/spring/spring-oauth2-utilities/src/main/java/org/opfab/springtools/configuration/oauth/OAuth2ReactiveConfiguration.java (hunk 8 above)
Comment (freddidierRTE) [reply tools/spring/spring-oauth2-utilities/src/main/java/org/opfab/springtools/configuration/oauth/OAuth2ReactiveConfiguration.java:11]: Fixed

File: tools/spring/spring-oauth2-utilities/src/main/java/org/opfab/springtools/configuration/oauth/jwt/groups/GroupsUtils.java (hunk 9 above)
Comment (freddidierRTE) [reply tools/spring/spring-oauth2-utilities/src/main/java/org/opfab/springtools/configuration/oauth/jwt/groups/GroupsUtils.java:14]: Fixed

**Pull request merged**